pkg/kubelet/images/image_gc_manager_test.go

```diff
@@ -448,7 +454,8 @@ func TestGarbageCollectImageNotOldEnough(t *testing.T) {
 
 	fakeClock := clock.NewFakeClock(time.Now())
 	t.Log(fakeClock.Now())
-	require.NoError(t, manager.detectImages(fakeClock.Now()))
+	_, err := manager.detectImages(fakeClock.Now())
+	require.NoError(t, err)
```

Comment:
i dont see any new unit tests added for the case you just fixed ?

Reply:
No more tests are needed, since we've already got `TestFreeSpaceImagesInUseContainersAreIgnored`. This test does test the function.

What this pr does is to not delete images used by running containers and prevent spam event messages like `image is being used by running container`.